### DISCUSSION TITLE
Status of cmorizers for reanalysis

### DISCUSSION THREAD

**Discussion by glpotter** [ESMValGroup/Community#124] [General]
@valeriupredoi , @zklaus, @bouweandela @hb326 This may be the wrong place to ask questions but, at the meeting this summer, we discussed a separate effort to cmorize reanalyses. Has anything happened? NASA is interested in contributing and storing data and may have some funding and staffing but I need to assure them that ESMValTool can be used. Does anybody have input?  Thanks, Jerry

**Comment by bouweandela** [comment]
Hi @glpotter, that is great news! We had a chat about this and agreed that @zklaus would contact you with more information. Did you receive word from him?

**Reply by glpotter** [reply]
Hi @bouweandela, I haven't heard anything from @zklaus.

**Reply by valeriupredoi** [reply]
Hi Jerry, have you checked your Spam folder, or did a search on 'Klaus'? Gmail has the tendency to 'think' these days, and hides some emails in weird places, but a search on sender will always find them

**Reply by zklaus** [reply]
Hi @glpotter, sorry to hear that. I did indeed send an email to your gmail address the October 16, 13:27 (CEST).
If that didn't work, could you send me an email?
